server/src/config/db.ts

```diff
@@ -0,0 +1,25 @@
+import dotenv from "dotenv";
+import mongoose from "mongoose";
+
+dotenv.config();
```

Comment:
I believe we dont need to do this again, if we have already configured it in app.ts